wlauto/external/uiauto/project.properties

```diff
@@ -11,4 +11,4 @@
 #proguard.config=${sdk.dir}/tools/proguard/proguard-android.txt:proguard-project.txt
 
 # Project target.
-target=android-17
+target=android-18
```

Comment:
Took on board your suggestions and after discussing with the rest of the team added some default test image names we will be using. These will be pushed at a later date in a separate dependency repo. At the moment we are trying to introduce these changes gradually.

I updated the .properties files to the new API requirement but I didn't rebuild the workloads since nothing in them has changed and they should run as before. Thank you for your input so far.